Cargo.toml

```diff
@@ -321,7 +321,7 @@ nix = { version = "0.24.1", default-features = false, features = ["socket", "sig
 
 [build-dependencies]
 prost-build = { version = "0.10.4", default-features = false, optional = true }
-tonic-build = { version = "0.7", default-features = false, features = ["transport", "prost", "compression"], optional = true }
+tonic-build = { git = "https://github.com/StephenWakely/tonic.git", branch = "stephen/compression_feature", default-features = false, features = ["transport", "prost"], optional = true }
```

Comment:
Could we leave a comment about why we are pinning and when we can unpin? Also can we pin to a SHA?